test/crv-staking/IncentivisedVotingLockup.ts

```diff
@@ -321,6 +343,61 @@ describe("IncentivisedVotingLockup", () => {
         const rbnRedeemer = redeemer.address;
         expect(rbnRedeemer).eq(await votingLockup.rbnRedeemer());
       });
+      it("sets contract stopped details", async () => {
+        const contractStopped = await votingLockup.contractStopped();
+        expect(contractStopped).eq(false);
+      });
+    });
+
+    describe("checking contract is stopped", () => {
+      it("fails when trying to lock as non-owner", async () => {
+        await expect(
+          votingLockup.connect(alice.signer).setContractStopped(true)
+        ).to.be.revertedWith("Ownable: caller is not the owner");
+      });
+
+      it("fails when trying to lock when contract is stopped", async () => {
+        await votingLockup
+          .connect(sa.fundManager.signer)
+          .setContractStopped(true);
+        await expect(
+          votingLockup
+            .connect(alice.signer)
+            .createLock(stakeAmt1, start.add(ONE_YEAR))
+        ).to.be.revertedWith("Contract is stopped");
+        await expect(
+          votingLockup.connect(alice.signer).increaseLockAmount(stakeAmt1)
+        ).to.be.revertedWith("Contract is stopped");
+        await expect(
+          votingLockup
+            .connect(alice.signer)
+            .increaseLockLength(start.add(ONE_YEAR))
+        ).to.be.revertedWith("Contract is stopped");
+      });
+      it("locks when contract is not stopped", async () => {
```

Comment:
Could we please add a newline between the `it` blocks so it's easier to read?

Reply:
yea. was expecting prettier to do it.